pages/bare_metal_cloud/dedicated_servers/replacing-user-password/guide.fr-fr.md

```diff
@@ -64,22 +64,29 @@ La commande exacte dépend du point de montage utilisé. Par exemple, si vous av
 chroot /mnt/
 ```
 
-### Étape 2 : réinitialiser le mot de passe de l'utilisateur
+### Étape 2 : identifier le(s) compte(s) utilisateur et réinitialiser le mot de passe
 
-Remarque : sur une distribution GNU/Linux, **une invite de mot de passe n'affiche pas vos entrées clavier**.
+Après avoir monté la partition et exécuté `chroot /mnt` (ou l'équivalent), vous disposez des privilèges **root** sur le système monté.
 
-Changez le mot de passe de l'utilisateur avec la commande suivante (remplacez `username` par le nom réel du compte utilisateur) :
+Avant de modifier un mot de passe, **identifiez les comptes existants** :
```

Comment:
Edit slightly to indicate "if necessary / if applicable" (if you dont know exactly which accounts to modify)
@benchbzh